pkg/mutation/schema/schema.go

```diff
@@ -186,3 +185,19 @@ func (db *DB) HasConflicts(id types.ID) bool {
 	defer db.mutex.RUnlock()
 	return db.conflicts[id]
 }
+
+func (db *DB) GetConflicts(id types.ID) map[types.ID]bool {
```

Comment:
There seem to be a number of uses of `map[types.ID]bool` that could be replaced with `mutationschema.IDSet`. It's obviously the same in functionality, but might make the function signatures portray greater semantic meaning.

Reply:
Done.